src/server/game/Entities/Player/Player.cpp

```diff
@@ -7138,8 +7138,8 @@ void Player::UpdateArea(uint32 newArea)
     else
         RemovePvpFlag(UNIT_BYTE2_FLAG_SANCTUARY);
 
-    uint32 const areaRestFlag = (GetTeam() == ALLIANCE) ? AREA_FLAG_REST_ZONE_ALLIANCE : AREA_FLAG_REST_ZONE_HORDE;
-    if (area && area->Flags[0] & areaRestFlag)
+    // uint32 const areaRestFlag = (GetTeam() == ALLIANCE) ? AREA_FLAG_REST_ZONE_ALLIANCE : AREA_FLAG_REST_ZONE_HORDE;
```

Comment:
I presume commenting out this line is temporary, maybe waiting to be included in a future commit (unless you didn't know that TC usually prefers to have lines removed instead of commented out).

Reply:
I was being lazy since Shauren wanted to comment on some faction balance things anyway, so I figured I'd just remove it after that.
I'll just do it now though

Reply:
Fair enough. Just as long as there are plans to change it later, that is OK too.